components/notion/actions/delete-block/delete-block.mjs

```diff
@@ -0,0 +1,29 @@
+import notion from "../../notion.app.mjs";
+import base from "../common/base-page-builder.mjs";
+
+export default {
+  ...base,
+  key: "notion-delete-block",
+  name: "Delete Block",
+  description: "Sets a Block object, including page blocks, to archived: true using the ID specified. [See the documentation](https://developers.notion.com/reference/delete-a-block)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    notion,
+    infoLabel: {
+      type: "alert",
+      alertType: "info",
+      content: "**Note:** In the Notion UI application, this moves the block to the \"Trash\" where it can still be accessed and restored.",
+    },
+    blockId: {
+      type: "string",
+      label: "Block ID",
+      description: "Block ID retrieved from the **Retrieve Page Content** action",
+    },
```

Comment:
Could this and other `blockId` props use the `pageId` propDefinition to list async options?

Reply:
Hi @michelle0927, I'm afraid this might return some errors, as these actions only delete/update blocks in the page content. It can't delete pages.